injitest/src/main/java/inji/pages/AddNewCardPage.java

```diff
@@ -19,8 +19,8 @@ public class AddNewCardPage extends BasePage{
     @iOSXCUITFindBy(accessibility = "goBack")
     private WebElement backButton;
     
-    @AndroidFindBy(accessibility = "issuer-ESignet")
-    @iOSXCUITFindBy(accessibility = "issuer-ESignet")
+    @AndroidFindBy(accessibility = "issuerHeading-ESignet")
+    @iOSXCUITFindBy(accessibility = "issuerHeading-ESignet")
     private WebElement downloadViaEsignet;
```

Comment:
This should be `issuerHeading-Mosip` right?

Reply:
updated